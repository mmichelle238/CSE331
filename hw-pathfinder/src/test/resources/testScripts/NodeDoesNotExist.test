# Find a path with Dijkstra's algorithm when
# the node doesn't exist.

CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B

FindPath simpleGraph C D