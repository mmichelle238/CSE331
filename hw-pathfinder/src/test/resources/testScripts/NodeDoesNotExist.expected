# Find a path with Dijkstra's algorithm when
# the node doesn't exist.

created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph

unknown node C
unknown node D